README.md

```diff
@@ -74,3 +76,45 @@ Results in this:
 ```
 hello-world!
 ```
+
+run-elm automatically detects whether `output` is of type `String` or `List String -> String`, so passing no extra arguments to the command will simply make `args` an empty list and will not produce an error.
+Similarly, if `output` is `String`, extra command arguments are simply ignored as redundant.
+
+### Customising output name
+
+It is possible to specify which constant or function you would like to output by adding `--output-name=myCustomConstantOrFunction` to the command:
```

Comment:
Specify a custom name for the output constant or function by adding `--output-name=myCustomConstantOrFunction` to the command: